.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches:
+    - main
+
+env:
+  DESTDIR: /tmp/build
+  LUA_LIB_DIR: /usr/local/openresty/lualib
+  CARGO_NET_GIT_FETCH_WITH_CLI: true
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    outputs:
+      published: ${{ steps.release.outputs.published }}
+      release-git-tag: ${{ steps.release.outputs.release-git-tag }}
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Release
+      id: release
+      uses: ahmadnassri/action-semantic-release@v2.1.10
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  artifacts:
+    needs: release
+    name: Create Release Artifacts
+    runs-on: ubuntu-latest
+    container: rustembedded/cross:${{ matrix.target }}
+    strategy:
+      matrix:
+        target:
+          - aarch64-unknown-linux-musl
+          - x86_64-unknown-linux-gnu
+          - x86_64-unknown-linux-musl
+          - aarch64-unknown-linux-gnu
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+        target: ${{ matrix.target }}
+        override: true
+    - name: Build and Install atc-router
+      env:
+        TARGET: ${{ matrix.target }}
+        RUSTFLAGS: '-C target-feature=-crt-static'
```

Comment:
I'm not sure if we still need this. As `cross` should automatically statically link against `musl` already. We do not generally want to statically link again Glibc.

Reply:
OK I'll try without :+1:

Reply:
I tried without and this is necessary for `aarch64-unknown-linux-musl`